scripts/build-wda.js

```diff
@@ -4,24 +4,36 @@ const B = require('bluebird');
 const {Simctl} = require('node-simctl');
 const {getSimulator} = require('appium-ios-simulator');
 const {logger} = require('appium/support');
+const yargs = require('yargs/yargs');
```

Comment:
is this module included into project dependencies?

Reply:
You're right, it's part of `package-lock.json`, no in project deps.
Can I add it?

Reply:
you can do it, although I usually prefer to avoid adding new dependencies unless there is a major reason to it. 
Check, for example, https://github.com/appium/WebDriverAgent/blob/9fd58b6c43ddeefd22ef379f8a92c867dfb4974b/Scripts/update-wda-version.js#L11